microsoft-edge/webview2/reference/HoloLens2/webview2-unity-plugin.md

```diff
@@ -0,0 +1,316 @@
+---
+title: WebView plugin for Unity API Reference (Preview)
+description: API Reference for Microsoft Mixed Reality WebView plugin for Unity.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: webview
+ms.date: 05/08/2023
+---
+# Mixed Reality WebView Plugin for Unity API Reference (Preview)
+The **Microsoft Mixed Reality WebView plugin for Unity** enables the integration of WebView2 functionality into your HoloLens 2 app.  This WebView plugin for Unity simplifies the integration of WebView2 functionality into your HoloLens 2 app by wrapping the WebView2 control, automatically handling rendering, and automatically directing input to the WebView2 control.
+
+This plugin also manages interop between Unity and WebView2, enabling communication between JavaScript and Unity via messages and events.
+
+This plugin exposes a subset of the functionality available via [CoreWebView2](/dotnet/api/microsoft.web.webview2.core.corewebview2).

```

Comment:
```suggestion
This plugin exposes a subset of the functionality that's available via [CoreWebView2](/dotnet/api/microsoft.web.webview2.core.corewebview2).
```
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/t/that-vs-which -- "Include that even if the sentence is clear without it. It helps to clarify the sentence structure."
https://learn.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/u/using-by-using-with -- "In instructions or technical content, use by using or that use to eliminate ambiguity."

Reply:
done locally, can close